scripts/publish_devto.py

```
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
```

Ruff (0.14.2)

Shebang is present but file is not executable

(EXE001)